app/routes/app/$mType/m/$laborMarketAddress/prereqs.tsx

```diff
@@ -46,8 +53,8 @@ export default function MarketplaceIdPrerequesites() {
                   You must hold this badge to review and score submissions on challenges
                 </p>
                 <div className="text-xs text-gray-500">MDAO S4 REVIEWER BADGE</div>
-                <div className="flex gap-2">
-                  <Avatar />
+                <div className="flex gap-2 items-center">
+                  <img src={maintainerURL} alt="" className="h-4 w-4" />
```

Comment:
I _think_ we want to use Avatar for these (even those these might look more natural in rectangular form